### DISCUSSION TITLE
Request for Assistance: Account Suspension

### DISCUSSION THREAD

**Discussion by parthgupta49** [community/community#118815] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear GitHub Support Team,

I hope this message finds you well. I am reaching out to you with regard to the recent suspension of my GitHub account, Parth-gupta49 associated with this email: [nbparthgupta4959@gmail.com] I received a notification stating that my access has been suspended due to a violation of your Terms of Service, but I am uncertain as to the specific reason for this action.
(I am unable to open even the contact page of GitHub with my old account that's why I am trying to ask for help with this account)

I would like to provide you with some context surrounding the events leading up to the suspension in hopes of clarifying the situation. Approximately a week or two ago, I noticed some unusual activity on my account. Specifically, a repository was created under my account without my authorization. Additionally, it appeared that I had started following a user with a username related to "hack," which was not an action initiated by me.

I would like to assure you that upon discovering the unauthorized activity on my account, I took immediate steps to secure it. Specifically, I changed the password for both my GitHub account and the associated email address, and I enabled two-factor authentication as an additional security measure. I am prepared to provide screenshots documenting these actions on the same day that the unauthorized repository was created, as evidence of my proactive response to safeguard my account. However, despite these precautionary measures, I was dismayed to discover that my account had been suspended upon attempting to access it a day or two later.

I want to assure you that the commits associated with my account, numbering around 206 or 204 prior to the incident, are entirely genuine and reflective of my own work. Any deviation from this count, such as the increase to around 208 commits that occurred when the unauthorized repository was created, was not of my doing. I took prompt action to rectify the situation by deleting the unauthorized repository as soon as I became aware of it.

My GitHub account serves as a crucial platform for me to showcase my work and collaborate with others in the community. Losing access to it has not only impacted my ability to share my projects but has also left me feeling unsettled about the security of my online presence.

I am writing to humbly request your assistance in reinstating my GitHub account at your earliest convenience. I am fully committed to cooperating with any investigations or providing further information that may be necessary to resolve this matter promptly. If there are any steps I can take to regain access to my account or if you require additional details from me, please do not hesitate to let me know.

Thank you for your attention to this matter. I eagerly await your response and sincerely hope for a positive resolution.

Warm regards,
Parth Gupta
Parth-gupta49 (My GitHub username - account which has been suspended)
![enabled_2factor-authentication](https://github.com/community/community/assets/166434636/cf317cfb-8cd7-485e-bd10-b651bf2ea444)
![unable-to-access-myAccount](https://github.com/community/community/assets/166434636/facf6305-4402-42db-8c2a-fb9d1fe48407)
![unableToContactSupport](https://github.com/community/community/assets/166434636/0d1ab0dc-a4ce-40cd-9c53-1ae5fa6e0072)
![changed-password](https://github.com/community/community/assets/166434636/9166e64c-a109-498e-b1a0-445c0a1d8b22)

**Comment by ebndev** [comment, marked as answer]
Hi [parthgupta49](https://github.com/parthgupta49),

Thank you for being a part of the GitHub Community!

This forum, **GitHub Community Discussions** focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. The best place to go for support regarding account-specific questions like this one is through opening a ticket on our [Support page](https://support.github.com/contact). It will be answered in the order it was received based on the support team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions or escalate your ticket, I am going to close this discussion.

We appreciate your understanding!
